torch/csrc/jit/tensorexpr/codegen.h

```diff
@@ -161,7 +175,8 @@ class RegisterCodeGen {
 TORCH_API std::unique_ptr<CodeGen> CreateCodeGen(
     const std::string& name,
     Stmt* stmt,
-    const std::vector<CodeGen::BufferArg>& params);
+    const std::vector<CodeGen::BufferArg>& params,
```

Comment:
Maybe for after this diff but I kind of think the CodeGen registry should be keyed on at::DeviceType instead of a random string.